factories/tola_management_models.py

```diff
@@ -0,0 +1,149 @@
+
+import json
+import datetime
+from factory.django import DjangoModelFactory
+from factory import lazy_attribute, LazyAttribute, SubFactory, Faker, Sequence, Iterator
+from factories.workflow_models import TolaUserFactory, CountryFactory, ProgramFactory, OrganizationFactory
+from factories.indicators_models import (
+    DisaggregationTypeFactory, DisaggregationLabelFactory, RFIndicatorFactory, LevelFactory, LevelTierFactory)
+from tola_management.models import (
+    CountryAdminAuditLog, ProgramAdminAuditLog, UserManagementAuditLog, OrganizationAdminAuditLog, ProgramAuditLog,
+    AuditLogRationaleSelection)
+
+
+def generate_mc_levels(obj, create, extracted, **kwargs):
+    from factories.indicators_models import LevelTierFactory
+    LevelTierFactory.build_mc_template(program=obj)
+
+
+class AuditLogRationaleSelectionFactory(DjangoModelFactory):
+    class Meta:
+        model = AuditLogRationaleSelection
+
+    other = Iterator([1,0,0,0,0,0,0,0])
+    adaptive_management = Iterator([0,1,0,0,0,0,0,0])
+    budget_realignment = Iterator([0,0,1,0,0,0,0,0])
+    changes_in_context = Iterator([0,0,0,1,0,0,0,0])
+    costed_extension = Iterator([0,0,0,0,1,0,0,0])
+    covid_19 = Iterator([0,0,0,0,0,1,0,0])
+    donor_requirement = Iterator([0,0,0,0,0,0,1,0])
+    implementation_delays = Iterator([0,0,0,0,0,0,0,1])
```

Comment:
this is cool!  Maybe a comment documenting what the Iterator does might be helpful for future maintainers - I imagine it falls out from the weird field that I used for this model, but I could see it being confusing for someone who hadn't written the base model

Reply:
Yep, done.